src/gatling/simulations/uk/gov/hmcts/reform/exui/performance/simulations/ExUI.scala

```diff
@@ -167,7 +175,32 @@ class ExUI extends Simulation {
 		.exec(EXUIMCLogin.manageCase_Logout)
   }
 
+	val recordedSimulationFRScn = scenario("FR").repeat(1)
```

Comment:
Let's give this a proper name please, recordedSimulationFRScn - what does that mean?!

Reply:
Done

Reply:
This hasn't changed? recordedSimulationFRScn - please make it fit with other naming conventions.

Reply:
Sorry, done now